src/data/graphics/items.h

```diff
@@ -1514,3 +1514,7 @@ const u32 gItemIconPalette_SteelMemory[] = INCBIN_U32("graphics/items/icon_palet
 
 const u32 gItemIcon_WaterMemory[] = INCBIN_U32("graphics/items/icons/water_memory.4bpp.lz");
 const u32 gItemIconPalette_WaterMemory[] = INCBIN_U32("graphics/items/icon_palettes/water_memory.gbapal.lz");
+
+const u32 gItemIconPalette_BugTMHM[] = INCBIN_U32("graphics/items/icon_palettes/bug_tm_hm.gbapal.lz");
+
```

Comment:
could you get rid of that new line there?

Reply:
I added it for consistency's sake. If you look at how the paths to the other TM/HM sprite palettes are defined above in the file, they're also separated from each other by an empty line.